src/Umbraco.Web.UI.Client/src/apps/backoffice/components/backoffice-header-logo.element.ts

```diff
@@ -57,7 +57,7 @@ export class UmbBackofficeHeaderLogoElement extends UmbLitElement {
 				<umb-popover-layout>
 					<div id="modal">
 						<img
-							src="/umbraco/backoffice/assets/umbraco_logo_blue.svg"
+							src="/umbraco/management/api/v1/security/back-office/graphics/logo-alternative"
```

Comment:
We need to add the server url here otherwise, it won't work on the Vite dev server.

Reply:
Why is the server address important now, and not before?

I have no idea how to had a mock endpoint in the client code.

Reply:
Before, the asset was shipped with the backoffice files. Now it's an endpoint, so it might reside on a different server. At least that is the case for the management api.

In the future, when we can load balance the backoffice, which server would serve the BackofficeController, the backoffice server or the api server? If the BackofficeController is always local to the backoffice files, then we don't need to prepend the server url.